Header header
robot_actions/ActionStatus status
annotated_map_builder/WaitActionGoal goal
std_msgs/Empty feedback